src/views/pages/authentication/auth-forms/AuthLogin.js

```diff
@@ -1,50 +1,30 @@
-import { useState } from "react";
```

Comment:
Remove changes for file from this PR, since they are not related to individual group page